### ISSUE TITLE
Generating a migration puts migration files in `_build` folder

### ISSUE THREAD

**Issue by ksmithut** [zookzook/elixir-mongodb-driver#160]
Steps to reproduce:

```sh
mix new mongo_test
cd mongo_test
```

Add `{:mongodb_driver, "~> 1.0.0"}` to the dependencies in `mix.exs`

```sh
mix do deps.get, deps.compile
mix mongo.gen.migration test
```

Outputs

```
* creating _build/dev/lib/mongodb_driver/priv/mongo/migrations
* creating _build/dev/lib/mongodb_driver/priv/mongo/migrations/20221129151911_test.exs
```

**Comment by zookzook** [comment]
Yes, I think that's right, because if you look at the _build file there should be a symbol link to your priv folder:

```
lrwxr-xr-x    1 micha  staff     16 25 Nov 09:21 priv -> ../../../../priv
```

And keep in mind, you need to configure the migration module.

**Comment by ksmithut** [comment]
So here's what I added to configure the migration module:

`config/config.exs`
```
import Config

config :mongodb_driver,
  migration: [
    otp_app: :mongo_test
  ]
```

Now when I run it, I get:

```
** (FunctionClauseError) no function clause matching in IO.chardata_to_string/1    
    
    The following arguments were given to IO.chardata_to_string/1:
    
        # 1
        {:error, :bad_name}
    
    Attempted function clauses (showing 2 out of 2):
    
        def chardata_to_string(string) when is_binary(string)
        def chardata_to_string(list) when is_list(list)
    
    (elixir 1.14.2) lib/io.ex:670: IO.chardata_to_string/1
    (elixir 1.14.2) lib/path.ex:548: Path.join/2
    (elixir 1.14.2) lib/path.ex:518: Path.join/1
    lib/tasks/gen/migration.ex:15: Mix.Tasks.Mongo.Gen.Migration.run/1
    (mix 1.14.2) lib/mix/task.ex:421: anonymous fn/3 in Mix.Task.run_task/4
    (mix 1.14.2) lib/mix/cli.ex:84: Mix.CLI.run_task/2
```

`:mongo_test` is the name of my mix app, did I name it wrong?

Also when running the steps before, it never generated a `priv` folder in my mix project. Is there some other standard step to set up the symlink? I've tried doing these from a generated phoenix project and just a plain mix project, and neither of them set up that symlink.

**Comment by zookzook** [comment]
This is a bug. The function :code.priv_dir/1 does not work like expected. I will fix it! Thanks for reporting!

**Comment by zookzook** [comment]
I see, the `:code.priv_dir/1 ` works only under some conditions and my local test accidentally hold that conditions :-)

**Comment by ksmithut** [comment]
Awesome, thanks for looking at it 😄

**Comment by zookzook** [comment]
I you like, you can try out the master branch. I replaced the function.

**Comment by ksmithut** [comment]
@zookzook That did the trick 👍 Thank you!

**Comment by zookzook** [comment]
Hey @ksmithut. I added support for options and multiple topologies to have full control of the migration. Check the README file.

**Comment by zookzook** [comment]
Yes! Maybe I find the time to draft a new release. For the time being you can use the master branch.

**Comment by zookzook** [comment]
Should be fixed in version 1.0.1!
